frontend/src/sass/_themes.scss

```diff
@@ -227,26 +232,44 @@ $httat-theme: (
 // Theme Init
 @include mat.all-component-themes($httat-theme);
 
-
 // Specific component overrides, pieces that are not in line with the general theming
 
 // Handle buttons appropriately, with respect to line-height
-.mat-raised-button, .mat-stroked-button, .mat-flat-button {
+.mat-raised-button,
+.mat-stroked-button,
+.mat-flat-button {
   padding: 0 1.15em;
-  margin: 0 .65em;
+  margin: 0 0.65em;
   min-width: 3em;
-  line-height: 36.4px
+  line-height: 36.4px;
 }
 
 .mat-standard-chip {
-  padding: .5em .85em;
+  padding: 0.5em 0.85em;
   min-height: 2.5em;
 }
 
 .material-icons {
   font-size: 24px;
-  font-family: 'Material Icons', 'Material Icons';  
+  font-family: "Material Icons", "Material Icons";
   .mat-badge-content {
-    font-family: 'Roboto';
+    font-family: "Roboto";
   }
+}
+
+// dark mode slider
+
+.dark-mode {
+  background-color: map-get($mat-dark-theme-background, background);
+  color: map-get($mat-dark-theme-foreground, text);
+}
+
+.content-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.toggle-container {
+  margin-left: auto;
```

Comment:
Please add `margin-right: 0.5em;` to `.toggle-container` ... that will remove the horizontal scroll bar.